app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/contact/ContactDetailsConfirmAddressController.scala

```diff
@@ -144,7 +144,7 @@ class ContactDetailsConfirmAddressController @Inject() (
                     addressLine2 = Some("Soho"),
                     townOrCity = "London",
                     postCode =
-                      partnershipDetails.generalPartnershipDetails.fold("W1T 2HN")(_.postcode)
+                      Some(partnershipDetails.generalPartnershipDetails.fold("W1T 2HN")(_.postcode))
```

Comment:
You can just "map" the details to extract an optional postcode

Reply:
👍